build/repo.targets

```diff
@@ -71,6 +76,11 @@
     <Exec Command="npm run build" WorkingDirectory="$(RepositoryRoot)clients/ts/FunctionalTests" IgnoreStandardErrorWarningFormat="true" />
   </Target>
 
+<Target Name="BuildJavaClient">
+   <Message Text="Building Java client" />
```

Comment:
Add `Importance="high"` like the others, otherwise it doesn't show at the default MSBuild verbosity.